registry/tests/templates.spec.ts

```diff
@@ -576,6 +576,49 @@ describe(`Tests ${example.url}`, () => {
         });
     });
 
+    describe('Partial Update', () => {
+        it("should not partial update any record if record doesn't exist", async () => {
+            const incorrect = { name: 123 };
+            await req
+                .patch(example.url + incorrect.name)
+                .send({ content: example.correct.content })
+                .expect(404, 'Not found');
+        });
+
+        it('should successfully partial update record', async () => {
+            await req.post(example.url).send(example.correct).expect(200);
+
+            const response = await req
+                .patch(example.url + example.correct.name)
+                .send({ content: example.updated.content })
+                .expect(200);
+            expect(response.body).deep.equal({
+                ...example.updated,
+                versionId: response.body.versionId,
+            });
+        });
+
+        it('should not delete localized versions of the template on partial update', async () => {
```

Comment:
Maybe even explicitly notify api user about passing localisedVersions is not supported?

Reply:
like in which way?

Reply:
validation in case locales were sent

Reply:
already implemented. Just added a test